test/distance_router_test.cc

```diff
@@ -155,7 +160,7 @@ class TShapeRoadRoutingTest : public ::testing::Test {
 };
 
 // Defines the test cases for the TShapeRoad where the start and end positions are on the very same Lane.
```

Comment:
> same Lane

Given the new name of the class, this could be confusing. Should it be "same Lane, which implicitly means they are in the same Segment"?

Reply:
[9a6e1ae](https://github.com/maliput/maliput_integration_tests/pull/84/commits/9a6e1aed7b757f0030f7ce6f8c439182228291cf)